packages/twenty-front/src/modules/workflow/search-variables/utils/extractVariableLabel.ts

```diff
@@ -0,0 +1,19 @@
+import { isDefined } from 'twenty-ui';
+
+export const extractVariableLabel = (rawVariable: string) => {
+  const variableWithoutBrackets = rawVariable.replace(
+    /\{\{([^{}]+)\}\}/g,
```

Comment:
can we use `REGEX_VARIABLE_TAG` here? I don't get the functional difference between the 2 regex

Reply:
This regex captures what's inside brackets. The other one also captures brackets. We could refactor it to use a single regex.